docs/installation/basic.md

```diff
@@ -91,3 +178,74 @@ python3 manage.py runserver
 ```
 
 * **Step 7** - Rejoice. Go to `localhost:5000` in your web browser to see the application live.
+
+
+## Flask-SocketIO development
+
+[Flask-SocketIO](https://flask-socketio.readthedocs.io/en/latest/) has been used in the project for displaying real-time notifications to the user. Although it's switched off by default. To integrate SocketIO you must set the `INTEGRATE_SOCKETIO` variable to `true` at bash.
+
+```bash
+export INTEGRATE_SOCKETIO="true"
+```
+
+The development server is the one that Flask ships with. It's based on Werkzeug and does not support WebSockets. If you try to run it, you'll get a RunTime error, something like: `You need to use the eventlet server. `.  To test real-time notifications, you must use the Gunicorn web server with eventlet worker class.
+
+If you've installed development requirements, you should have both `gunicorn` and `eventlet` installed. To run application on port 5000, execute the following instead of `python3 manage.py runserver`:
+
+```bash
+gunicorn app.instance:app --worker-class eventlet -w 1 --bind 0.0.0.0:5000 --reload
+```
+
+`-w` specifies the number of worker classes to be used. `--reload` is used for development environments, so the server is restarted if any of the application python files change.
+
+Now you should be able to access the website at `localhost:5000`.
```

Comment:
This section is obsolete. Now, we don't have support for real time notifications

Reply:
Done